### PR TITLE
sysctl: add German translation

### PR DESCRIPTION
**Pull request by k-stz**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For the word "Filehandle" I opted to use both the German and English translation side-by-side. I couldn't find enough credible sources for a German translation and the rest of the tldr pages doesn't use the word either.

This is my first PR in a long time, I'm a bit rusty, please feel free to point out "obvious" mistakes, I want to do more PR during #hacktoberfest. :)

 - **Version of the command being documented (if known):** not known

### CODE REVIEW

File: pages.de/linux/sysctl.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# sysctl
+
+> Kernelparameter zur Laufzeit auflisten und ändern. Als Parameter stehen die Werte unter `/proc/sys/` zur Verfügung (in Unterordner organisiert).
```
Comment (marchersimon) [inline pages.de/linux/sysctl.md:3]: This does seem useful, but we sholudn't add anything which isn't in the English page. But you can of course open a PR to add it there.
```suggestion
> Laufzeit-Kernelparameter auflisten und ändern.
```

File: pages.de/linux/sysctl.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Liste alle verfügbaren Kernelparameter mit ihren Werten auf:
+
+`sysctl -a`
+
+- Setze einen veränderbaren Kernelparameter:
+
+`sysctl -w {{sektion.tunable}}={{Wert}}`
```
Comment (marchersimon) [inline pages.de/linux/sysctl.md:11]:
```suggestion
`sysctl -w {{sektion.tunable}}={{wert}}`
```

File: pages.de/linux/sysctl.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Aktuell geöffnete Datei-Handle (Filehandle) abfragen:
```
Comment (marchersimon) [inline pages.de/linux/sysctl.md:13]:
```suggestion
- Frage aktuell geöffnete Datei-Handler ab:
```

File: pages.de/linux/sysctl.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+`sysctl fs.file-nr`
+
+- Frage maximale Anzahl geöffneter Dateien ab:
```
Comment (marchersimon) [inline pages.de/linux/sysctl.md:17]:
```suggestion
- Frage die maximale Anzahl geöffneter Dateien ab:
```

File: pages.de/linux/sysctl.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+`sysctl fs.file-max`
+
+- Änderungen aus der `/etc/sysctl.conf` Datei übernehmen:
```
Comment (marchersimon) [inline pages.de/linux/sysctl.md:21]:
```suggestion
- Übernimm Änderungen aus der `/etc/sysctl.conf` Datei:
```

**Review by marchersimon — APPROVED** [review]
Thanks

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**